### PR TITLE
Setting up affiliated pages and zfit

### PR DESCRIPTION
**Pull request by mayou36**
First approach to the affiliated pages.

Should we have bullet points as in projectusers as well or subtitles as the explanation may be larger? @eduardo-rodrigues

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
Thanks very much @mayou36 for this first PR on affiliated packages! I will inline some comments.

File: pages/affiliated.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Affiliated Packages
```
Comment (eduardo-rodrigues) [inline pages/affiliated.md]:
```suggestion
title: Affiliated packages
```

File: pages/affiliated.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+permalink: /affiliated
+nav_order: 6
+---
+
+
+Affiliated Packages {#affiliated_packages}
```
Comment (eduardo-rodrigues) [inline pages/affiliated.md]:
```suggestion
Affiliated packages {#affiliated_packages}
```

File: pages/affiliated.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+====================
+
+In the following, projects that work closely together with scikit-hep are described. They extend the Python ecosystem in HEP and remain, due to their size and scope, generally independent of scikit-hep.
```
Comment (eduardo-rodrigues) [inline pages/affiliated.md]: You should use "Scikit-HEP" for the project (2 occurrences in these 2 lines).

File: pages/affiliated.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+
+zfit: scalable pythonic fitting
+-------------------------------
+
+[![img-zfit]][zfit]
+
+The [zfit](https://zfit.github.io/zfit/) project brings together the different efforts of fitting in Python by providing a stable model
+fitting API and workflow. This comes together with an implementation of the fitting library that is based on TensorFlow, a deep learning framework built for high performance computing on modern architectures. While traditional functionalities such as loading ROOT-files or using the Minuit minimizer are integrated (thanks to scikit-hep packages), it is completely independent of the ROOT project and has no dependency on it.
```
Comment (eduardo-rodrigues) [inline pages/affiliated.md]: Avoid "... together" again. How about "The implementation ..." instead of "This comes together with an implementation ..."?

Comment (eduardo-rodrigues) [inline pages/affiliated.md]: Add links for TensorFlow, ROOT and Minuit.

scikit-hep -> Scikit-HEP

File: pages/affiliated.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+The core of zfit is a class-based approach for model building, composition and customization capabilities.
+
+The project consists of additional libraries which extend the capabilities of the core.
```
Comment (eduardo-rodrigues) [inline pages/affiliated.md]:
```suggestion
The project consists of additional libraries extending the capabilities of the core package.
```

File: pages/affiliated.md (hunk 3 above)
Comment (mayou36) [reply pages/affiliated.md]: done

File: pages/affiliated.md (hunk 4 above)
Comment (mayou36) [reply pages/affiliated.md]: yes, true

Comment (mayou36) [reply pages/affiliated.md]: done

**Comment by eduardo-rodrigues** [comment]
This is certainly good for a first version (more on affiliated packages/projects to be added asap).

Thanks a lot @mayou36!

**Pull request merged**